libs/ui/select/brain/src/lib/brn-select.component.ts

```diff
@@ -309,11 +294,16 @@ export class BrnSelectComponent<T = unknown>
 	}
 
 	protected _canOpen(): boolean {
-		return !this.isExpanded() && !this._disabled() && this.options()?.length > 0;
+		return !this.isExpanded() && !this.disabled() && this.options()?.length > 0;
 	}
 
 	private _moveFocusToCDKList(): void {
-		setTimeout(() => this.selectContent.focusList());
+		setTimeout(() => {
+			const content = this.selectContent();
+			if (content) {
+				content.focusList();
+			}
+		});
```

Comment:
```suggestion
		setTimeout(() => this.selectContent()?.focusList?.())
```
Thoughts on using optional chaining?